docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -107,15 +107,25 @@ is used.
       * ``ZSTD``
       * ``GZIP``
     - ``ZSTD``
+  * - ``iceberg.use-file-size-from-metadata``
+    - Some Iceberg writers populate incorrect file sizes in the metadata. When
+      this property is set to false, Trino ignores the stored values and fetches
+      them with a getFileStatus call. This means an additional call per split,
+      so it is recommended for a Trino admin to fix the metadata, rather than
+      relying on this property for too long.
+    - ``true``
   * - ``iceberg.max-partitions-per-writer``
     - Maximum number of partitions handled per writer.
     - 100
   * - ``iceberg.target-max-file-size``
-    - Target maximum size of written files; the actual size may be larger
+    - Target maximum size of written files; the actual size may be larger.
     - ``1GB``
   * - ``iceberg.unique-table-location``
-    - Use randomized, unique table locations
+    - Use randomized, unique table locations.
     - ``false``
+  * - ``iceberg.dynamic-filtering.wait-timeout``
+    - Duration to wait for completion of dynamic filters during split generation.
```

Comment:
"Maximum duration", we may wait less than that time set if the DF is ready first